e2e/Dockerfile-e2e

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+
+RUN apt update && apt upgrade -y
+
+ENV NODE_VERSION=20.10.0
+RUN apt install -y curl
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+ENV NVM_DIR=/root/.nvm
+RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
+RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
+RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
+ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
+
+ENV CI=1
+
+COPY . ./
+
+RUN npm install -g yarn && yarn
+RUN yarn playwright install --with-deps
+
+ENTRYPOINT [ "yarn", "playwright", "test" ]
```

Comment:
Does this use the `playwright.config.ts` file by default or do we need to pass the `--config <file>` flag?

Reply:
It does :)